components/support/locale/src/main/java/mozilla/components/support/locale/LocaleManager.kt

```diff
@@ -48,6 +50,33 @@ object LocaleManager {
         return currentLocale
     }
 
+    /**
+     * Change the current locale to the system defined one.
+     * [getCurrentLocale] will return null
+     *
+     * After calling this function, to visualize the locale changes you have to make sure all your visible activities
+     * get recreated. If your app is using the single activity approach, this will be trivial just call
+     * [AppCompatActivity.recreate]. On the other hand, if you have multiple activity this could be tricky, one
+     * alternative could be restarting your application process see https://github.com/JakeWharton/ProcessPhoenix
+     *
+     */
+    fun resetToSystemDefault(context: Context) {
```

Comment:
Would you mind adding a test for this section? If you need any help just let me know eager to help!

Reply:
Added, turns out there's a [config annotation for robolectric](http://robolectric.org/device-configuration/), which solved my testing problem.

Reply:
Awesome :)